lib/do_image/do_update.js

```diff
@@ -10,12 +10,15 @@
  * `triton image update ...`
  */
 
+var jsprim = require('jsprim');
 var vasync = require('vasync');
 
 var common = require('../common');
 var errors = require('../errors');
 var UPDATE_IMAGE_FIELDS =
     require('../cloudapi2').CloudApi.prototype.UPDATE_IMAGE_FIELDS;
+var updateImageFields = jsprim.deepCopy(UPDATE_IMAGE_FIELDS);
```

Comment:
Maybe add a comment here - someone will wonder why this is so.

Reply:
Sure, done!